### ISSUE TITLE
[Taxon]: Suggested change to Source.Voltage.DC

### ISSUE THREAD

**Issue by MarkKuster** [NCSLI-MII/measurand-taxonomy#44]
### Contact Details

mjk@ieee.org

### What is the process type?

Source

### Measurand

see existing description--no change at this time

### Kind of quantity

Voltage

### Does the measurand require a special token?

None

### Parameters

New (optional) Parameters:
        Temperature[.Ambient]
        Ratio.Humidity.Relative[.Ambient]
        Current.DC[.Input]

### Taxon Name

Source.Voltage.DC

**Comment by ryanmwhitephd** [comment]
Link to existing definition: https://github.com/NCSLI-MII/measurand-taxonomy/blob/main/source/Measure_Voltage_DC.xml

**Comment by JonBartholomew** [comment]
This looks good I think I understand it but I'm not sure the linking is correct below

 <mtc:ReferenceUrl>
        <mtc:name>KCDB</mtc:name>
        <mtc:url>bipm.org/kcdb</mtc:url>
      </mtc:ReferenceUrl>
      <mtc:CategoryTag>
        <mtc:name>High Voltage DC</mtc:name>
        <mtc:value>8.1</mtc:value>
      </mtc:CategoryTag>
    </mtc:Reference>

I'd pick the following 3 categories first 

1.1.1 Single values: standard cell, solid state voltage standard
1.1.2 Low value ranges (below or equal to 10 V): DC voltage source, multifunction calibrator
1.1.3 Intermediate values (above 10 V to 1100 V): DC voltage source, multifunction calibrator

In section 8 we would have

8.1.4 Average value of DC voltage. 8.1.4 would be linked to source and measure is this a problem

**Comment by MarkKuster** [comment]
Yes, the existing linkage doesn't capture the whole picture or even the primary category. The new schema coming to fruition through the Flask app will capture all the KCDB linkages.

**Comment by ryanmwhitephd** [comment]
Currently a blocker is specifying parameters. 

Temperature & Relative Humidity parameters should specify ambient values as opposed to the UUT's temperature, for example.

**Comment by ryanmwhitephd** [comment]
Is it sufficient to include as parameters, and reference the quantities/aspects?
Temperature
RelativeHumidity
Current

For Current, Voltage, Power, the taxon includes AC or DC, is it neccesary to include Current.DC?

**Comment by SchwartzMichael** [comment]
They could be added as Optional
